components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/Oauth2ScopeConstants.java

```diff
@@ -30,6 +30,7 @@
     public static final String SCOPE_TYPE_OAUTH2 = "OAUTH2";
     public static final String SCOPE_TYPE_OIDC = "OIDC";
 
+    public static final String AUTHORISED_INTERNAL_SCOPES_PARAM_NAME = "AUTHORISED_INTERNAL_SCOPES";
```

Comment:
AUTHORISED or AUTHORIZED ??

Reply:
Both are correct I guess. Z is used in US English. :D

Reply:
Which one do we usually use?

Reply:
In docs we use "Z"